snippets/formToObject.md

```diff
@@ -0,0 +1,25 @@
+### formToObject
+
+Serializes a form into an object.
+
+First we transform the `form` into `FormData`, then we convert it into an `array` and from the `array` we collect an `object`
```

Comment:
Explain a bit more in-depth the methods used, see existing snippets for reference.

Reply:
i'm update description